doc-requirements.txt

```diff
@@ -12,3 +12,4 @@ ipython
 mock
 colorspacious
 pillow
+sphinx-gallery
```

Comment:
unfortunately, we can't as this file is meant to be used by pip, and graphviz is not pip installable.